src/utils/hooks/useProjects.ts

```diff
@@ -0,0 +1,15 @@
+import { modelToGroupVersionKind, ProjectModel } from '@kubevirt-ui/kubevirt-api/console';
+import { K8sResourceCommon, useK8sWatchResource } from '@openshift-console/dynamic-plugin-sdk';
+
+type UseProjects = () => string[];
+
+const useProjects: UseProjects = () => {
```

Comment:
please check if we dont have this kind of hook

Reply:
We have `useProjectsAndPVCs` and `useProjectsAndDataSources`, but we don't have a hook for just projects. I suggest we keep this `useProjects` and use it inside of the other hooks.